runtime/interface.cpp

```diff
@@ -601,6 +605,10 @@ void f$fastcgi_finish_request(int64_t exit_code) {
       break;
     }
     case QUERY_TYPE_HTTP: {
+      // Run custom header handler before body processing
+      if (header_custom_handler_function) {
```

Comment:
What if register function like this? Now it will break http response and call callback twice
```php
function foo() {
  echo "1";
  flush(); 
}
```